guides/source/action_view_overview.md

```diff
@@ -517,43 +556,50 @@ CAUTION: Only keyword arguments are supported. Defining positional or block argu
 Layouts
 -------
 
-Layouts can be used to render a common view template around the results of Rails controller actions. Typically, a Rails application will have a couple of layouts that pages will be rendered within. For example, a site might have one layout for a logged in user and another for the marketing or sales side of the site. The logged in user layout might include top-level navigation that should be present across many controller actions. The sales layout for a SaaS app might include top-level navigation for things like "Pricing" and "Contact Us" pages. You would expect each layout to have a different look and feel. You can read about layouts in more detail in the [Layouts and Rendering in Rails](layouts_and_rendering.html) guide.
+Layouts can be used to render a common view template around the results of Rails controller actions. A Rails application can have multiple layouts that pages can be rendered within. 
+
+For example, an application might have one layout for a logged in user and another for the marketing part of the site. The logged in user layout might include top-level navigation that should be present across many controller actions. The sales layout for a SaaS app might include top-level navigation for things like "Pricing" and "Contact Us" pages. Different layouts typically have a different header and footer content. 
+
+To find the layout for the current controller action, Rails first looks for a file in `app/views/layouts` with the same base name as the controller. For example, rendering actions from the PhotosController class will `use app/views/layouts/photos.html.erb` (or `app/views/layouts/photos.builder`). 
+
+If such a controller-specific layout does not exist, Rails will use `app/views/layouts/application.html.erb` (or `app/views/layouts/application.builder`). Rails also provides more ways to assign specific layouts to individual controllers and actions. See [Layouts and Rendering guide](layouts_and_rendering.html#finding-layouts) for more.
+
+You can learn more about layouts in general in the [Layouts and Rendering in Rails](layouts_and_rendering.html) guide.
```

Comment:
The two consecutive links to the layouts and rendering guide here seemed a bit odd on first reading... maybe we just leave this paragraph, and eliminate the previous "see... for more" portion?

Reply:
sure, yeah! I was trying to link to a specific section. But on second thought, I will refrain from that until layout and rendering sections are finalized. Then might come in there to add more links if needed as a follow on.